features/spicy-protocol-analyzer/testing/tests/standalone.spicy

```diff
@@ -3,5 +3,11 @@
 # @TEST-EXEC: spicyc ${DIST}/analyzer/@ANALYZER_LOWER@.spicy -j -d -o @ANALYZER_LOWER@.hlto
 #
 # TODO: Add/adapt standalone parsing tests here.
+#
+#       Note: If you cannot easily test your parser using "printf", consider
+#       using spicy-driver's batch mode instead to test with application
+#       payload extracted from a PCAP trace. See the tutorial in the Spicy
+#       documentation for more on how to create and use such batch files.
```

Comment:
Let's directly add a link (to a dedicated version if we want deal with link rot).

Slight formatting nit: many editors highlight `NOTE` like `TODO` or `FIXME`, also let's get rid of the leading spaces.

```suggestion
# NOTE: If you cannot easily test your parser using "printf", consider
# using spicy-driver's batch mode instead to test with application
# payload extracted from a PCAP trace. See the tutorial in the Spicy
# documentation for more on how to create and use such batch files,
# https://docs.zeek.org/projects/spicy/en/v1.11.3/toolchain.html#batch-input.
```

Reply:
That's the wrong link though, not the tutorial, which I think is the better place. I'll update once the new tutorial has been merged.